sys/include/net/coap.h

```diff
@@ -139,13 +144,21 @@ extern "C" {
  * @anchor  net_coap_format
  * @{
  */
-#define COAP_FORMAT_TEXT         (0)
-#define COAP_FORMAT_LINK        (40)
-#define COAP_FORMAT_XML         (41)
-#define COAP_FORMAT_OCTET       (42)
-#define COAP_FORMAT_EXI         (47)
-#define COAP_FORMAT_JSON        (50)
-#define COAP_FORMAT_CBOR        (60)
+#define COAP_FORMAT_TEXT                      (0)
+#define COAP_FORMAT_LINK                     (40)
+#define COAP_FORMAT_XML                      (41)
+#define COAP_FORMAT_OCTET                    (42)
+#define COAP_FORMAT_EXI                      (47)
+#define COAP_FORMAT_JSON                     (50)
+#define COAP_FORMAT_CBOR                     (60)
+#define COAP_FORMAT_APPLICATION_SENML_JSON  (110)
```

Comment:
I see in RFC 7252 that the name of the format for JSON, for example, is "application/json", yet we don't use the 'application' part in the macro. Do we need to add 'application' for the new macros? It makes the names significantly longer, and it seems like 'COAP_FORMAT_xxx' already eliminates any ambiguity. I'm not really schooled on format names, so maybe I'm missing something though.

Reply:
I don't mind dropping the `_APPLICATION` part. We could also shorten it to `_APP`, but I see no real harm in dropping it completely.